app/views/pages/home.html.haml

```diff
@@ -170,27 +170,27 @@
         %h1 Get Involved
         %ul
           %li.link
-            #{link_to 'GitHub', 'https://github.com/hackedu'}
+            #{link_to 'GitHub', 'https://github.com/hackclub'}
           %li.link
-            = link_to 'Join us on IRC', 'https://kiwiirc.com/client/irc.freenode.net/?nick=Hacker%7C?#hackedu'
+            = link_to 'Join us on IRC', 'https://kiwiirc.com/client/irc.freenode.net/?nick=Hacker%7C?#hackclub'
       .small-12.medium-4.columns.nav
         %h1 Social
         %ul
           %li.link
             #{link_to 'Facebook', 'https://www.facebook.com/pages/hackEDU/741805665870458'}
           %li.link
-            #{link_to 'Twitter', 'https://twitter.com/starthackedu'}
+            #{link_to 'Twitter', 'https://twitter.com/starthackclub'}
       .small-12.medium-4.columns.nav
         %h1 Contact Us
         %ul
-          %li.link #{mail_to 'team@hackedu.us'}
+          %li.link #{mail_to 'team@hackclub.io'}
```

Comment:
`%... \#{expression}` can be written without interpolation as `%...= expression`